src/messages.cs

```diff
@@ -0,0 +1,261 @@
+//
+// Messages bindings
+//
+// Authors:
+//	Vincent Dondain <vincent@xamarin.com>
+//
+// Copyright 2016 Xamarin Inc. All rights reserved.
+//
+
+using System;
+using XamCore.CoreFoundation;
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+
+#if !MONOMAC
+namespace XamCore.Messages {
+
+	[Introduced (PlatformName.iOS, 10, 0)]
+	[Native]
+	public enum MSMessagesAppPresentationStyle : nuint
+	{
+		Compact,
+		Expanded
+	}
+
+	[Introduced (PlatformName.iOS, 10, 0)]
+	[Native]
+	public enum MSStickerSize : nint
+	{
+		Small,
+		Regular,
+		Large
+	}
+
+	[Introduced (PlatformName.iOS, 10, 0)]
+	[BaseType (typeof(UIViewController))]
+	interface MSMessagesAppViewController
+	{
+		[NullAllowed, Export ("activeConversation", ArgumentSemantic.Strong)]
+		MSConversation ActiveConversation { get; }
+
+		[Export ("presentationStyle", ArgumentSemantic.Assign)]
+		MSMessagesAppPresentationStyle PresentationStyle { get; }
+
+		[Export ("requestPresentationStyle:")]
+		void RequestPresentationStyle (MSMessagesAppPresentationStyle presentationStyle);
```

Comment:
Maybe `Request` ? The parameter (enum) type will already make it clear it's a presentation style when reading the code (so we can drop the duplication without losing context)